src/QueryDescription/index.d.ts

```diff
@@ -49,10 +49,29 @@ declare module '@nozbe/watermelondb/QueryDescription' {
     left: ColumnName
     comparison: Comparison
   }
-  export type Condition = Where | On
+  export interface SortBy {
+    type: 'sortBy'
+    sortColumn: ColumnName
+    sortOrder: SortOrder
+  }
+  export type SortOrder =
+    | 'asc'
+    | 'desc'
+  export interface Take {
+    type: 'take'
+    count: number
+  }
+  export interface Skip = {
+    type: 'skip'
+    count: number
+  }
+  export type Condition = Where | On | SortBy | Take | Skip
```

Comment:
do you have an idea how to rename Condition to something that would better describe the extended role?

Reply:
In SQL, things like `WHERE`, `ORDER BY` and `LIMIT` are collectively called "SQL Clauses".
Maybe call it "QueryClause"? Or "QuerySpecifier"?

Reply:
Yeah, `Clause` sounds reasonable!